README.md

```diff
@@ -1,6 +1,23 @@
 # Welcome to Crowbar Angular Prototype
 [![Build Status](https://travis-ci.org/crowbar/crowbar-ui.svg?branch=master)](https://travis-ci.org/crowbar/crowbar-ui)
 
+### Setting up a development environment with Docker
+
+1. make sure `docker` is installed and the `docker` daemon is running.
+2. build the image from the [Dockerfile](Dockerfile):
+```
+docker build -t crowbar-ui .
+```
+3. run the container:
+```
+docker run -d --name "crowbar-ui_container" -v $(pwd)/assets:/crowbar-ui/assets -p 3000:3000 crowbar-ui
+```
+4. run the `gulp` watchdog:
+```
+docker exec -ti crowbar-ui_container gulp
+```
+5. work on the assets folder in your local git clone and the changes will be immediately visible in the browser.
```

Comment:
i see. how do you usually access the coverage report then? is it served via express.js as well?

you're probably right about the objective, it is not really clear yet. it should just be as easy as possible to quickly start developing, throwing away, and rebuilding ;-) and a container can be great for that. 

but anyways I guess this is still much WIP, so I'll try around a bit (currently with your suggestion `supervisord`)